CHANGELOG/v1.19.1.md

```diff
@@ -0,0 +1,24 @@
+## Changes by Kind
+
+### Bug Fix
+
+- Update ASO CRDs to v2.11.0 ([#5516](https://github.com/kubernetes-sigs/cluster-api-provider-azure/pull/5516), [@k8s-infra-cherrypick-robot](https://github.com/k8s-infra-cherrypick-robot))
+
+### Documentation
+
+- Updating the roadmap to align with our current timeline and future milestones ([#5514](https://github.com/kubernetes-sigs/cluster-api-provider-azure/pull/5514), [@k8s-infra-cherrypick-robot](https://github.com/k8s-infra-cherrypick-robot))
```

Comment:
I'd probably vote to manually remove this documentation section, as it isn't really relevant to the v1.19.1 bits that we're releasing

Reply:
Probably innocuous _Documentation addition_ imo if we look at this patch release as any other patch release. 
But happy to remove it nonetheless!